### DISCUSSION TITLE
Vercel not fetching from Dall E

### DISCUSSION THREAD

**Discussion by DarkoKrezic** [neuefische/questions#622] [Web]
Hi, 
i am currently having the problem that vercel is not fetching the images from Dall-E. Locally everything is working alright. Here is the branch i am working on [Branch](https://github.com/DarkoKrezic/capstone-projekt/pull/15). When i want to create a new Story using the Storyteller button the story can not be generated and i am not being pushed to the EditStoryPage, probably because of the missing data.

**Comment by DarkoKrezic** [comment]
Found a solution for the next 14days using vercel pro trial version.

**Reply by pixelass** [reply]
vercel has a limit at 10 seconds for free tier and 1 minute for pro. that means that the pro version might also fail if the dall-e response takes to long.

You can look into edge functions: 

- https://vercel.com/docs/concepts/functions/edge-functions
- https://nextjs.org/docs/app/building-your-application/rendering/edge-and-nodejs-runtimes
